src/adapters/refract/transformResource.coffee

```diff
@@ -40,9 +40,9 @@ module.exports = (resourceElement, location, options) ->
     resourceParameters = getUriParameters(_.get(resourceElement, 'attributes.hrefVariables'), options)
     actionParameters = getUriParameters(_.get(transitionElement, 'attributes.hrefVariables'), options)
 
-    attributes = _.dataStructures(transitionElement)
-    attributes = if _.isEmpty(attributes) then _.dataStructures(resourceElement) else attributes[0]
+    attributes = _.dataStructures(resourceElement)
```

Comment:
@pksunkara Could you provide a blueprint or source refract document that demonstrates this changed? I'm not sure what this actually change and I can't see any modified behaviour in my testing.